doc/howto/container_gpu_passthrough_with_docker.md

```diff
@@ -0,0 +1,147 @@
+(container-gpu-passthrough-with-docker)=
+# How to pass an NVIDIA GPU to a container
+
+If you have an NVIDIA GPU (either discrete (dGPU) or integrated (iGPU)) and you want to pass the runtime libraries and configuration installed on your host to your container, you should add a {ref}`LXD GPU device <devices-gpu>`.
+Consider the following scenario:
+
+Your host is an NVIDIA single board computer that has a Tegra SoC with an iGPU, and you have the Tegra SDK installed on the host. You want to create a LXD container and run an application inside the container using the iGPU as a compute backend. You want to run this application inside a Docker container (or another OCI-compliant runtime).
+To achieve this, complete the following steps:
+
+1. Initialize a LXD container with {config:option}`instance-security:security.nesting` enabled (needed for running a Docker container inside a LXD container):
+
+        lxc init ubuntu:24.04 t1 --config security.nesting=true
+
+1. Add the GPU device to your container:
+
+    - If you want to do an iGPU pass-through:
+
+          lxc config device add t1 igpu0 gpu gputype=physical id=nvidia.com/igpu=0
+
+    - If you want to do a dGPU pass-through:
+
+          lxc config device add t1 gpu0 gpu gputype=physical id=nvidia.com/gpu=0
+
+After adding the device, let's try to run a basic [MNIST](https://en.wikipedia.org/wiki/MNIST_database) inference job inside our LXD container.
+
+1. Create a `cloud-init` script that installs the Docker runtime, the [NVIDIA Container Toolkit](https://github.com/NVIDIA/nvidia-container-toolkit), and a script to run a test [TensorRT](https://github.com/NVIDIA/TensorRT) workload:
+
+        #cloud-config
```

Comment:
In general, as this is a howto, I think it would be useful to the reader to add comments to the cloud-init file explaining what each step is doing and why?

Reply:
fixed